deps/rabbitmq_prometheus/src/rabbit_prometheus_handler.erl

```diff
@@ -163,6 +163,11 @@ put_filtering_options_into_process_dictionary(Request) ->
             put(prometheus_mf_filter, Fs);
         _ -> ok
     end,
+    case rabbit_mgmt_agent_config:get_env(filter_aggregated_queue_metrics_pattern) of
```

Comment:
This creates an implicit dependency on the management agent plugin. We should use a separate Prometheus-specific app key instead.